packages/focusvisible/src/useFocusVisible.js

```diff
@@ -325,15 +300,5 @@ export function useFocusVisible({
 
       clearTimeout(hadFocusVisibleRecentlyTimeout.current);
     };
-  }, [
-    addInitialPointerMoveListeners,
-    onBlur,
-    onFocus,
-    onKeyDown,
-    onPointerDown,
-    onVisibilityChange,
-    relativeDocument,
-    removeInitialPointerMoveListeners,
-    scope
-  ]);
```

Comment:
By moving `addInitialPointerMoveListeners`,	`onBlur`,	`onFocus`,	`onKeyDown`,	`onPointerDown`,	`onVisibilityChange`,	and	`removeInitialPointerMoveListeners`	into the `useEffect`, they do not need to be memoized.